packages/replayio/src/utils/authentication/config.ts

```diff
@@ -1,4 +1,4 @@
-import { getReplayPath } from "../getReplayPath";
+import { getReplayPath } from "@replay-cli/shared/getReplayPath";
```

Comment:
This is the only thing I already started to inline as part of this PR. You can inspect the `replayio`'s output to see how this looks like. In particular this file gets transpiled to:
```js
'use strict';

var getReplayPath = require('../../shared/dist/getReplayPath.js');

const authClientId = process.env.REPLAY_AUTH_CLIENT_ID || "4FvFnJJW4XlnUyrXQF8zOLw6vNAH1MAo";
const authHost = process.env.REPLAY_AUTH_HOST || "webreplay.us.auth0.com";
const cachedAuthPath = getReplayPath.getReplayPath("profile", "auth.json");

exports.authClientId = authClientId;
exports.authHost = authHost;
exports.cachedAuthPath = cachedAuthPath;
```